stringbasic/String.cpp

```diff
@@ -0,0 +1,58 @@
+#include "String.h"
+
+#include <cstring>
+#include <iostream>
+
+String::String()
+{}
+
+String::String(const char* string) {
+	changeString(string);
+}
+
+String::String(const String& other) {
+	(*this) = other;
```

Comment:
Is this the desired behavior?
Test this code to make sure you don't have a double free bug (im pretty sure you do 😢  )